test/unit_tests/modules/classes/BreakpointHandler-spec.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+var chai = require( 'chai' );
+var expect = chai.expect;
+var jsdom = require( 'mocha-jsdom' );
+var sinon = require( 'sinon' );
+
+var BASE_LOC = '../../../../cfgov/v1/preprocessed/js/';
+var BreakpointHandler =
+require( BASE_LOC + 'modules/classes/BreakpointHandler' );
+var args;
+
+beforeEach( function() {
+  args = {
+    enter:      function() {},
+    leave:      function() {},
+    breakpoint: 599
+  };
+} );
+
+describe( 'BreakpointHandler', function() {
+
+  jsdom( {
+    created: function( error, win ) {
+      if ( error ) {
+        console.log( error ); // eslint-disable-line no-console, no-inline-comments, max-len
+      }
+
+      var resizeEvent = win.document.createEvent( 'Event' );
+      resizeEvent.initEvent( 'resize', true, true );
+
+      win.resizeTo = function( width, height ) {
+        this.innerWidth = this.outerWidth = width;
+        this.innerHeight = this.outerHeight = height;
+        win.dispatchEvent( resizeEvent );
+      };
+    }
+  } );
+
+  it( 'should throw an error if passed incomplete arguments', function() {
+    var errorTxt = 'BreakpointHandler constructor requires arguments!';
+    function createBreakpointInstance() {
+      return new BreakpointHandler();
+    }
+
+    expect( createBreakpointInstance ).to.throw( errorTxt );
+  } );
+
+  it( 'should compute window width properly on older browsers', function() {
+    function createBreakpointInstance() {
+      return new BreakpointHandler( args );
+    }
+    delete window.innerWidth;
+
+    expect( createBreakpointInstance ).to.not.throw( Error );
+    window.innerWidth = window.outerWidth;
+  } );
+
+  it( 'should correctly create BreakpointHandler instances', function() {
+    var breakpointHandler = new BreakpointHandler( args );
+    expect( breakpointHandler.watchWindowResize )
+    .to.be.an.instanceof( Function );
+    expect( breakpointHandler.handleViewportChange )
+    .to.be.an.instanceof( Function );
+    expect( breakpointHandler.testBreakpoint )
+    .to.be.an.instanceof( Function );
+
+    expect( breakpointHandler.match ).to.be.false;
+    expect( breakpointHandler.type === 'max' ).to.be.true;
```

Comment:
This is fine, but could do `expect( breakpointHandler.type ).to.equal( 'max' );` also.